CHANGELOG.rst

```diff
@@ -1034,34 +1075,26 @@ Cliconf
 New Modules
 -----------
 
-- ios_acl_interfaces - ACL interfaces resource module
-- ios_acls - ACLs resource module
-- ios_banner - Manage multiline banners on Cisco IOS devices
-- ios_bgp - Configure global BGP protocol settings on Cisco IOS.
-- ios_command - Run commands on remote devices running Cisco IOS
-- ios_config - Manage Cisco IOS configuration sections
-- ios_facts - Collect facts from remote devices running Cisco IOS
-- ios_interface - (deprecated, removed after 2022-06-01) Manage Interface on Cisco IOS network devices
```

Comment:
this got removed!

Reply:
The anstsibull-changelog tool always removes it and makes it revert back to the older form of the file.

Reply:
In that case for now you can revert this manually